UE/Tests/Application/ApplicationTestSuite.cpp

```diff
@@ -162,29 +162,54 @@ TEST_F(ApplicationConnectedTestSuite, shallHandleShowSms)
     objectUnderTest.handleShowSms(index);
 }
 
-TEST_F(ApplicationConnectedTestSuite, shallHandleCallRequest)
+TEST_F(ApplicationConnectedTestSuite, shallHandleReceiveCallRequest)
 {
     EXPECT_CALL(userPortMock, showCallRequest(_));
     EXPECT_CALL(timerPortMock, startTimer(30000ms));
-    objectUnderTest.handleCallRequest(PHONE_NUMBER);
+    objectUnderTest.handleReceiveCallRequest(PHONE_NUMBER);
 }
 
-TEST_F(ApplicationConnectedTestSuite, shallHandleCallAccepted)
+TEST_F(ApplicationConnectedTestSuite, shallHandleSendCallAccepted)
 {
     constexpr common::PhoneNumber to{200};
     EXPECT_CALL(btsPortMock, sendCallAccepted(to));
     EXPECT_CALL(userPortMock, showTalking());
     EXPECT_CALL(timerPortMock, stopTimer());
-    objectUnderTest.handleCallAccept(to);
+    objectUnderTest.handleSendCallAccept(to);
 }
 
-TEST_F(ApplicationConnectedTestSuite, shallHandleCallDropped)
+TEST_F(ApplicationConnectedTestSuite, shallHandleSendCallDropped)
 {
     constexpr common::PhoneNumber to{200};
     EXPECT_CALL(timerPortMock, stopTimer());
     EXPECT_CALL(btsPortMock, sendCallDropped(to));
     EXPECT_CALL(userPortMock, showConnected());
-    objectUnderTest.handleCallDrop(to);
+    objectUnderTest.handleSendCallDrop(to);
+}
+
+TEST_F(ApplicationConnectedTestSuite, shallHandleStartDial)
+{
+    EXPECT_CALL(userPortMock, showEnterPhoneNumber());
+    objectUnderTest.handleStartDial();
+}
+
+TEST_F(ApplicationConnectedTestSuite, shallHandleSendCallRequest)
+{
+    common::PhoneNumber myPhoneNumber = PHONE_NUMBER;
+    constexpr common::PhoneNumber enteredPhoneNumber{200};
+    EXPECT_CALL(btsPortMock, sendCallRequest(myPhoneNumber,enteredPhoneNumber));
+    EXPECT_CALL(timerPortMock, startTimer(30000ms));
```

Comment:
shouldn't this be 60s?

Reply:
Yeah, it should. I did not read it, this information was in the failure scenario and i missed it.